pkg/gds/admin_test.go

```diff
@@ -401,7 +401,7 @@ func (s *gdsTestSuite) TestSummary() {
 		VASPsCount:           14,
 		PendingRegistrations: 5,
 		ContactsCount:        39,
-		VerifiedContacts:     28,
+		VerifiedContacts:     24,
```

Comment:
Should we change some of these checks to be less specific?

Reply:
We could; alternatively we could add error messages to the tests to the effect of this to help us more quickly figure out what's going on if the tests fail
```golang
require.Len(t, actual.ContactsCount, 39, "contact count is incorrect; have the fixtures changed?")
```